.github/workflows/release.yml

```diff
@@ -96,37 +97,61 @@ jobs:
   build-native:
     runs-on: ubuntu-latest
     needs: create-release
+    strategy:
+      matrix:
+        include:
+          # Build with these flags
+          - name: debian
+            container: ubuntu-dev:20
+          - name: rpm
+            container: fedora:30
     container:
-      image: ghcr.io/romange/ubuntu-dev:20
+      image: ghcr.io/romange/${{ matrix.container }}
     steps:
     - uses: actions/checkout@v3
       with:
         submodules: true
     - name: Configure
       run: |
-          apt update && apt install -y debhelper moreutils pip jq
+          if [ -f /etc/redhat-release ]; then
+            dnf install -y rpm-build glibc-static libstdc++-static
```

Comment:
Why do we need `glibc-static`? I thought we're only changing libgcc and libstdc++.

Reply:
will remove